sets/treeset/iterator.go

```diff
@@ -6,7 +6,7 @@ package treeset
 
 import (
 	"github.com/emirpasic/gods/containers"
-	rbt "github.com/emirpasic/gods/trees/redblacktree"
+	"github.com/spewspews/gods/trees"
```

Comment:
these all need to be changed, but only after this gets merged in. Otherwise it won't build against the current state of the base repo